key_action_map.cpp

```diff
@@ -455,8 +455,9 @@ void test_key_action_map()//!OCLINT tests can be many
     assert(kam.to_key(action_type::turn_right) == sf::Keyboard::D);
     assert(kam.to_key(action_type::accelerate_forward) == sf::Keyboard::W);
     assert(kam.to_key(action_type::accelerate_backward) == sf::Keyboard::S);
-    assert(kam.to_key(action_type::shoot) == sf::Keyboard::Q);
-    assert(kam.to_key(action_type::shoot_stun_rocket) == sf::Keyboard::E);
+    // Note that this line is commented out due to the need of minimal viable game, ref to (644)
+    //assert(kam.to_key(action_type::none) == sf::Keyboard::Q);
```

Comment:
```suggestion
    assert(kam.to_key(action_type::none) == sf::Keyboard::Q);
```

Reply:
Hey, I tried again and found out why I commented this out, the reason is if not then the test will fail, I don't quite get it but this is a necessary step, and I kept it because I haven't figured out the importance of this test, so I just kept 😸

Reply:
I've figured out why :)
In this test, we are testing a function (`key_action_map::to_key`) that given a type of action, returns the key that is mapped to it.
Here, we mapped two keys to the same action type (`action_type::none`), so I suppose the second one (E) overrides the first (Q) when called. E is returned both times.

Reply:
I feel there is no point of testing which key "nothing" corresponds to, so I suggest simply removing both lines for now, and resurrect them (mapped to stun rocket and rocket) after we are done with the MVG version.

Reply:
removed these tests